src/index.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+require('regenerator-runtime/runtime');
+require('source-map-support').install();
+const Once = require('./utils/Once');
+const _ = require('lodash');
+const bugsnag = require('bugsnag');
+const uuidv1 = require('uuid/v1');
+const LambdaContext = require('./utils/LambdaContext');
+const Config = require('./Config');
+
+module.exports = {
+  initialize: function() {
+    this.once = new Once(async () => {
+      // Set up configuration dict
+      this.config = new Config([
+        require('./DataSources/Environment'),
+        require('./DataSources/Credstash'),
+        require('./DataSources/Clients'),
+        require('./DataSources/Logging'),
+        require('./DataSources/Replacer')
+      ]);
```

Comment:
Could this take place in Config, so that it's ready to go when you import it? Then it would not have to be added as a property.

Reply:
My goal here was really to keep Config a reusable "dictionary loaded from plugins", so that it could be used in other projects.

Reply:
So is the idea that the Config module will eventually have its own repo/npm module?

Reply:
It kinda already did...config-lambda.  I pulled it and credstash-lambda into classy-lambda-common with this PR because it just turns out that trying to refactor something that's spread over 3 npm modules is a nightmare, and I didn't feel the code was complex enough for everything to have its own little module.

What I'd like to do is move everything from classy-lambda-common into a "classy-common" or "classy-util" or "classy-pay-util" kind of package, and have both generic tools like this as well as non-lambda tools (I've talked about building up a Currency class for example that handles abstracting away currency math, etc.).  Think of it like "Lodash but for Classy-specific problems" but built like Lodash, in a way so that others can use and contribute if they like.